### PR TITLE
Bump android sdk to 34

### PR DESCRIPTION
**Pull request by acoutts**

### CODE REVIEW

File: packages/flutter_libphonenumber_android/android/build.gradle [hunk 1]
```diff
@@ -2,14 +2,14 @@ group 'com.bottlepay.flutter_libphonenumber'
 version '1.0-SNAPSHOT'
 
 buildscript {
-    ext.kotlin_version = '1.6.10'
+    ext.kotlin_version = '1.8.21'
```
Comment (Inventrohyder) [inline packages/flutter_libphonenumber_android/android/build.gradle:5]: Since we are already in here, we can set it as `1.8.22` as of June 23, 2023

**Review by Inventrohyder — APPROVED** [review]

File: packages/flutter_libphonenumber_android/android/build.gradle (hunk 1 above)
Comment (acoutts) [reply packages/flutter_libphonenumber_android/android/build.gradle:5]: Happy to do this if it's still relevant.

**Review by joco1412 — CHANGES_REQUESTED** [review]
add namespace in build.gradle for AGP 8.0


 namespace 'com.example.libphonenumber'

**Comment by alpha2048** [comment]
Is this pull request moving forward?
I'm having trouble with old Android settings.

**Comment by acoutts** [comment]
Published in android 1.1.0

**Pull request merged**